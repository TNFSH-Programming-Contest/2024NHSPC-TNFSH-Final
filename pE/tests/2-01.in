1 7513874 8209460 7632053 3572141
55
