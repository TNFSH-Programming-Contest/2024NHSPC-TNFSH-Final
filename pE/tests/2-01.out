876856245598944
